Content.Shared/AlertLevel/ChangeAlertLevel.cs

```diff
@@ -1,8 +1,17 @@
+using Robust.Shared.Audio;
 using Robust.Shared.Serialization;
 
 namespace Content.Shared.ChangeAlertLevel
```

Comment:
file scope

Reply:
it needs this for the audio noise :(

Reply:
someone saying file scope means change this

```cs
namespace Among.Us
{
    public class Sus {}
}
```

to this

```cs
namespace Among.Us;

public class Sus {}
```

its purely cosmetic and has no impact on audio